### PR TITLE
Adding sample3.txt

### PR DESCRIPTION
**Pull request by johnorillo**
Hi @leifsyliongka , please review the sample.txt thanks

### CODE REVIEW

File: sample3.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Section 1:
```
Comment (leifsyliongka) [inline sample3.txt]: This seems wrong...

**Review by leifsyliongka — COMMENTED** [review]
Nyaammm...

File: sample3.txt (hunk 1 above)
Comment (johnorillo) [reply sample3.txt]: Why can you please elaborate

Comment (leifsyliongka) [reply sample3.txt]: Shouldn't this be section 2?

File: sample3.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Section 2:
```
Comment (leifsyliongka) [inline sample3.txt]: Nyamm... Tama na..

File: sample3.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+
+First sample for branch 3
```
Comment (leifsyliongka) [inline sample3.txt:3]: Change this.

**Review by leifsyliongka — CHANGES_REQUESTED** [review]

**Review by leifsyliongka — APPROVED** [review]

**Pull request merged**